backend/movies/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class Movie(models.Model):
+    id = models.AutoField(primary_key=True)
+    title = models.CharField(max_length=30, blank=False)
+    genre = models.CharField(max_length=30, blank=False)
+
+    def create_movie(data):
+        movie = Movie(**data)
+        movie.set_title(movie.title)
```

Comment:
je nepotrebno cini mi se a i ne postoji metoda nigde, setovace title ako se isto zove u data i na modelu

movie = Movie(**data)
movie.save() 
to bi trebalo da odradi posao

isto i linija ispod